### PR TITLE
Move parallel testing examples

### PR DESCRIPTION
**Pull request by c3ccl3ston**

### CODE REVIEW

**Comment by c3ccl3ston** [comment]
@nadvolod - I'm not 100% sure about this but I think this is a clear way of showing the individual examples that were under parallel-testing. I don't mind reworking it so please let me know your thoughts.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ everything that you need to get started with web, mobile, visual, functional and
 ## 🥇Most Popular
 *  [Web automation best practices framework with multiple testing strategies. Crafted by industry experts with decades of experience.](/best-practice/src/test/java/com/saucedemo/tests/)
 *  [Quick start test, Junit 4](./java11/selenium3.junit4.examples/src/test/java/com/onboarding/Module1JunitTest.java)
-*  [Quick start test, TestNg](./selenium-testng-examples/src/test/java/TestNgTest.java)
+*  [Quick start test, TestNg](./selenium-testng-examples/src/test/java/com.saucedemo.TestNgTest.java)
```
Comment (nadvolod) [inline README.md:19]: @c3ccl3ston i think this link won't work, you'll need to do like this `...java/com/saucedemo/PerformanceExampleTests.java`

File: selenium-testng-examples/README.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+- [Cross Browser/Platform in Parallel w/ TestNG](./src/test/java/com/saucedemo/CrossBrowserPlatformTest.java)
+
+## How to run tests
+
+### Single Browser Test in Parallel
+```bash
+cd selenium-testng-examples
+mvn test -Dtest=com.saucedemo.ParallelSingleBrowserTest
+```
+<details>
```
Comment (nadvolod) [inline selenium-testng-examples/README.md:16]: @c3ccl3ston love the use of this tag!

**Review by nadvolod — COMMENTED** [review]
@c3ccl3ston great work! I think there's just one minor change to make.

File: README.md (hunk 1 above)
Comment (c3ccl3ston) [reply README.md]: Dang! I missed one

**Review by nadvolod — APPROVED** [review]

**Comment by c3ccl3ston** [comment]
@nadvolod should I go ahead and merge this pull request as you have approved it now, or is that something that you will continue to do? Just want to know what is expected. Thanks!

**Pull request merged**